frontend/utils/src/types/index.ts

```diff
@@ -38,11 +38,10 @@ export interface StreamlitWindowObject {
   // These window variables are used so that some deployments of Streamlit can
   // edit the index.html served to the client so that a Streamlit server at an
   // origin different from where the frontend static assets are served can be
-  // set. Note that we also need to have a separate `declare global` block here
-  // rather than adding to the one in App.tsx as these also need to be
-  // accessible within this package when no app exists.
+  // set.
   BACKEND_BASE_URL?: string
   HOST_CONFIG_BASE_URL?: string
+  DOCUMENT_BASE_URL?: string
```

Comment:
nit: maybe add one short comment for every option here describing what it does. E.g. I think it might not be fully clear what the difference between `BACKEND_BASE_URL` and `DOCUMENT_BASE_URL` is.

Reply:
Done! I also renamed `DOCUMENT_BASE_URL` -> `MAIN_PAGE_BASE_URL` since I think that name is much more self-explanatory.